src/LibraryManager/Providers/Unpkg/UnpkgLibraryGroup.cs

```diff
@@ -24,7 +25,9 @@ public UnpkgLibraryGroup(string displayName, string description = null)
 
             if (npmPackageInfo != null)
             {
-                return npmPackageInfo.Versions.Select(semanticVersion => semanticVersion.ToString()).ToArray();
+                return npmPackageInfo.Versions
```

Comment:
We should at ToList() to this Linq, so that it captures a snapshot of the results.